config/migrations/2024/20240626161836-beeindigd-status.sparql

```diff
@@ -0,0 +1,19 @@
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX cs: <http://data.vlaanderen.be/id/conceptscheme/>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX ext:	<http://mu.semte.ch/vocabularies/ext/>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX msc: <http://data.vlaanderen.be/id/concept/MandatarisStatusCode/>
+
+INSERT DATA {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    msc:b8866fa2-d61c-4e3d-afaf-8a29eaaa7fb9
+	    skos:topConceptOf	cs:MandatarisStatusCode;
+	    skos:inScheme	cs:MandatarisStatusCode;
+      rdf:type
+        skos:Concept,
+        ext:MandatarisStatusCode;
+	    skos:prefLabel "Beëindigd";
```

Comment:
Not sure if we set the other concept values to be a capital letter or not and if it matters.
![image](https://github.com/lblod/app-lokaal-mandatenbeheer/assets/36266973/17d8e19a-3cf5-467c-bf38-fcc61f45fdb6)

Reply:
Nevermind just saw it here ;p https://data.vlaanderen.be/doc/concept/MandatarisStatusCode?filter%5Bschema%5D%5Bid%5D=conceptscheme-MandatarisStatusCode

Reply:
Yeah the others are capitalised as well, as they should be, otherwise this might give troubles with sorting etc.